sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -14635,4 +14635,462 @@ public void testLatestByOnStringColumnWithoutMaxBytesSpecified()
             new Object[] {"abc", defaultString, "def", defaultString, "def", defaultString}
         ));
   }
+
+  @Test
+  public void testEquiJoin()
```

Comment:
Also, can you please use descriptive names for the test, where they represent what the test is primarily supposed to be verifying, or documents the regression why the test was added. 

For example: `testEquiJoinWhereLeftHandIsConstant`